reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -28,6 +28,7 @@ import com.here.ort.model.RemoteArtifact
 import com.here.ort.model.RuleViolation
 import com.here.ort.model.ScanResult
 import com.here.ort.model.ScanSummary
+import com.here.ort.model.VcsInfo
```

Comment:
Commit message: It should be an "empty placeholder *package*", or?

Reply:
Correct, fixed.